double-great/great-great-jekyll-theme
# Jekyll config

Fixes #35 by:
- changing from `jekyll-seo` to `jekyll-seo-tag` in config.yml
- providing minimal Gemfile required to use without GitHub Pages gem

Also excludes `.jekyll-cache` directory, created when using canonical Jekyll gem.

**To test**

Build and serve using GitHub Pages gem
1. `git fetch`
2. `git checkout jekyll-config`
3. `bundle install`
4. `bundle exec jekyll serve`
5. Confirm there are no build or serve issues
6. <kbd>⌃</kbd> + <kbd>c</kbd>

Build and serve using Jekyll gem and specific plugins
1. Edit `Gemfile` 
2. Comment out `gem "github-pages"  , group: :jekyll_plugins` line
3. Uncomment lines 10-17
4. Save changes
5. `bundle install`
6. `bundle exec jekyll serve`
7. Confirm there are no build or serve issues
8. <kbd>⌃</kbd> + <kbd>c</kbd>
